client/verta/verta/environment/__init__.py

```diff
@@ -9,6 +9,6 @@
 
 
 documentation.reassign_module(
-    [Docker, Python],
+    [Python],
```

Comment:
This hides `Docker` from our documentation, since it doesn't have any user-facing purpose yet (and therefore would just be confusing).